src/core/components/AppGetStarted.vue

```diff
@@ -6,20 +6,29 @@
         <div class="d-flex align-center px-6">
           <div>
             <h1 class="white--text mb-3">Ready to explore Ethereum?</h1>
-            <p class="white--text ma-0">
-              Still don't know what MEW is?
+            <div align="left" justify="space-around" class="mt-5">
```

Comment:
wow had no idea you can do `align` like that on a div 

not sure if justify is doing anything though i can't find any docs about it, if you could link me that'd be great!

Reply:
Oh yeah that can be removed, it's only for some Vue components.